be/dtm0_log.c

```diff
@@ -637,42 +652,93 @@ M0_INTERNAL int m0_be_dtm0_plog_prune(struct m0_be_dtm0_log    *log,
 M0_INTERNAL void m0_be_dtm0_log_pmsg_post(struct m0_be_dtm0_log *log,
```

Comment:
This function implementation has nothing related to BE.
So, the "_be_" in the name is not necessary. Maybe this function can be removed, and caller should call `m0_dtm0_dtx_pmsg_post()` directly.

Reply:
Fixed.